Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uss

```diff
@@ -0,0 +1,27 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import "../common.uss";
+
+Image#HelpPageReportIssueIcon {
+    --unity-image: resource("Images/Icons/GitHubIcon");
+}
+
+Image#HelpPageDocumentationIcon {
+    --unity-image: resource("Images/Icons/AWSIcon");
+}
+
+Image#HelpPageVideoTutorialIcon {
+    --unity-image: resource("Images/Icons/VideoIcon");
+}
+
+Image#HelpPageForumIcon {
+    --unity-image: resource("Images/Icons/PersonIcon");
+}
```

Comment:
These two images appear small in the UI compared to the GitHub icon

Reply:
fixed, they had incorrect class

Reply:
It looks like we updated in the wrong direction by making the other icons small instead of these icons larger

Reply:
Fixed!